### ISSUE TITLE
SC2029's wording implies a Warning level, actually an error

### ISSUE THREAD

**Issue by juan-borla** [koalaman/shellcheck#859]
SC2029 gives an error on intentional client-side-expanded variables used in remote SSH commands.  The wording implies the error should actually be a Warning.  The wording of the page on SC2029 even says
`If you do want your string expanded on the client side, you can safely ignore this message.`

#### For bugs
- Rule Id (if any, e.g. SC1000): SC2029
- My shellcheck version (`shellcheck --version` or "online"): 0.4.5
- [X] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/usr/bin/env bash
source "variables.sh"
ssh "$deploy_user"@"$server" "rm -rf \"$final_root_directory/feeds/feeds\""

```

#### Here's what shellcheck currently says:
Error: SC2029: Note that, unescaped, this expands on the client side.

#### Here's what I wanted or expected to see:
Warning: SC2029: Note that, unescaped, this expands on the client side.

**Comment by koalaman** [comment]
Shellcheck currently had four fuzzy categories: error, warning, info and style. This issue should be classified as an info message. Where are you seeing it as an error?

**Comment by juan-borla** [comment]
You're correct; The issue appeared when using an addon for Atom.io.  Testing using the direct script, the message was properly labeled.  Apologies.  Closing the issue.
